resources/assets/coffee/react/_components/beatmap-icon.coffee

```diff
@@ -37,7 +37,9 @@ el = React.createElement
 
     showTitle = @props.showTitle && !@props.overrideVersion?
 
+    mode = if beatmap.convert then 'osu' else beatmap.mode
```

Comment:
that's why i personally view the idea of showing converts as *separate* diffs as quite bad, as technically they are just standard maps played in different modes, and the only thing that differentiates them is scores.

maybe a better solution would be to add a mode picker to the scoreboard when viewing a standard diff? (i think it is like that on old site right now)